packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.tsx

```diff
@@ -0,0 +1,1512 @@
+    "The date you have selected is on a weekday that is not allowed. Please select another date.";
+
+  /**
+   * If `true`, dates in the future are not allowed. A validation message will appear if a date in the future is entered.
+   */
+  @Prop() disableFuture?: boolean = false;
+
+  /**
+   * The text to display as the validation message when `disableFuture` is `true` and a date in the future is entered.
+   */
+  @Prop() disableFutureMessage?: string =
+    "Dates in the future are not allowed. Please select a date in the past.";
+
+  /**
+   * If `true`, dates in the past are not allowed. A validation message will appear if a date in the past is entered.
+   */
+  @Prop() disablePast?: boolean = false;
+
+  /**
+   * The text to display as the validation message when `disablePast` is `true` and a date in the past is entered.
+   */
+  @Prop() disablePastMessage?: string =
+    "Dates in the past are not allowed. Please select a date in the future.";
+
+  /**
+   * The helper text that will be displayed for additional field guidance. This will default to the text "Use format" along with the `dateFormat` value.
+   */
+  @Prop() helperText?: string;
+
+  /**
+   * The ID for the input field. The default will be an automatically generated value.
+   */
+  @Prop() inputId?: string;
+
+  /**
+   * The label for the date input.
+   */
+  @Prop() label!: string;
+
+  /**
+   * The latest date that will be allowed. The value can be in any format supported as `dateFormat`, in ISO 8601 date string format (`yyyy-mm-dd`) or as a JavaScript `Date` object.
+   * The value of this prop is ignored if `disableFuture` is set to `true`.
+   */
+  @Prop() max?: string | Date = "";
+
+  @Watch("max")
+  watchMaxHandler(): void {
+    if (this.disableFuture) {
+      this.maxDate = new Date();
+    } else {
+      this.maxDate = createDate(this.max, this.dateFormat);
+    }
+  }
+
+  /**
+   * The earliest date that will be allowed. The value can be in any format supported as `dateFormat`, in ISO 8601 date string format (`yyyy-mm-dd`) or as a JavaScript `Date` object.
+   * The value of this prop is ignored if `disablePast` is set to `true`.
+   */
+  @Prop() min?: string | Date = "";
+
+  @Watch("min")
+  watchMinHandler(): void {
+    if (this.disablePast) {
+      this.minDate = new Date();
+    } else {
+      this.minDate = createDate(this.min, this.dateFormat);
+    }
+  }
+
+  /**
+   * The name of the control, which is submitted with the form data.
+   */
+  @Prop() name?: string;
+
+  /**
+   * The date visible when the calendar opens. Used if no date is currently selected.
+   * The value can be in any format supported as `dateFormat`, in ISO 8601 date string format (`yyyy-mm-dd`) or as a JavaScript `Date` object.
+   */
+  @Prop() openAtDate: string | Date = "";
+
+  /**
+   * If `true`, the input will require a value.
+   */
+  @Prop() required?: boolean = false;
+
+  /**
+   * Determines whether days outside the current month are rendered.
```

Comment:
Can we update this description to match the style of other boolean prop descriptions e.g "if true..."